wfl/Models/FoodOrder.cs

```diff
@@ -46,4 +46,9 @@ public void addMenuSelection(MenuSelection selection)
             }
         }
     }
+
+    public class StatusOptionsRepresentation
```

Comment:
@jspringer2020 was this a design you worked with @TNagaraja on regarding a need to return a representation of status options to assist with filtering?